tests/Unit/Admin/ProductBlocksServiceTest.php

```diff
@@ -193,13 +193,35 @@ public function test_register_merchant_center_setup_is_not_complete() {
 		$this->is_mc_setup_complete = false;
 
 		$this->simple_anchor_group->get_root_template()
-			->expects( $this->exactly( 0 ) )
+			->expects( $this->exactly( 1 ) )
 			->method( 'add_group' );
 
+		$this->simple['group']
+			->expects( $this->exactly( 1 ) )
+			->method( 'add_block' )
+			->with(
+				[
+					'id'         => 'google-listings-and-ads-product-onboarding-prompt',
+					'blockName'  => 'google-listings-and-ads/product-onboarding-prompt',
+					'attributes' => [ 'startUrl' => 'http://example.org/wp-admin/admin.php?page=wc-admin&path=/google/start' ],
+				]
+			);
+
 		$this->variation_anchor_group->get_root_template()
-			->expects( $this->exactly( 0 ) )
+			->expects( $this->exactly( 1 ) )
```

Comment:
💅  Another simple version for `$this->exactly( 1 )`  and `$this->exactly( 0 )` that we normally use is `$this->once()` and `$this->never()`. Not against using `$this->exactly()` just mentioning it in case you didn't know.

Reply:
Thanks for the information. I chose `exactly` because it's relatively effortless to maintain.